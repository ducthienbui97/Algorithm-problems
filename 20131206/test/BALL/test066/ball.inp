8 256869 928787 15351 233493
222317 884162 745265 13561 259425 327561 14470 798734 997829 909180 7584 206978 447845 497603 101147 865539 340125 191969 284380 719872 335723 208305 720075 186232 297035 164750 619223 78053 152542 736639 658405 224810 531762 478841 377645 873235 670762 14471 225064 209411 811035 305205 769961 882355 55117 403537 178362 93270 715844 815665 47767 458325 868185 649637 902316 788153 274560 292423 240980 890906 35668 274019 75244 780454 
987669 136212 452728 815725 626141 661750 866788 402584 27089 302787 11124 187022 212438 776516 267550 124711 326269 242009 950767 325675 444046 233069 203496 149248 82764 231976 491568 392390 487468 209592 251383 249310 468935 686331 701266 487049 191003 855459 114194 261553 215526 551411 727538 377602 388272 404481 700323 581589 114439 149945 770697 638198 795579 566478 758919 658622 734671 590290 640546 611372 982047 37939 446287 235814 
975562 178839 114490 496419 888425 88002 548818 313571 771931 900224 518870 808662 325116 387725 306859 660256 723037 142681 463143 110974 89771 653691 329315 597788 515714 754591 760955 442426 257640 928277 341186 457876 483118 907938 515052 834076 514338 736243 331349 185807 797922 943286 356118 73946 26656 955936 302601 309335 407819 19555 466761 568814 481259 620440 857556 76782 763050 901824 395716 302648 346224 8945 664176 349841 
901688 62403 903962 137239 410337 301138 380674 360304 738494 82589 678761 676766 775851 655362 939835 305657 344682 639219 439524 315648 620993 154394 940016 21842 582093 114092 18883 28342 391090 398520 227322 893242 574123 87737 224524 266099 938041 903251 528196 135766 888937 347596 70377 92993 146770 931071 898151 343937 93925 645984 727152 413343 92324 385907 315068 654287 342993 616514 49854 290636 86766 345201 215991 683120 156507 689125 375870 59187 485625 293240 338952 413136 750325 239403 416089 713608 884664 298166 940260 620135 327341 212451 116976 14251 58047 266113 810862 127152 491137 909152 797578 585818 936205 273916 555541 449815 498339 904029 160885 168137 80044 66815 502188 320860 79258 541822 446962 678579 8429 824308 206917 404737 439700 592477 163592 318043 298233 22573 842590 699594 66011 782788 361008 321067 970304 45657 155846 236549 304425 467060 775226 448058 675967 701449 362409 140930 857050 23061 952091 764524 923454 862791 152830 901379 843044 842968 527465 996442 743843 18192 815191 406848 201366 390384 701731 187648 272853 257542 133172 397559 795435 36879 327792 913708 856476 208238 805093 406339 525272 455493 442477 865581 809486 791812 389458 884592 988139 524310 478943 743645 266885 398107 249702 982333 776346 85084 400470 704153 206881 594886 853966 448840 34998 432782 719434 936211 176591 907216 873513 949612 200081 599354 261848 893795 929072 220490 734151 331098 238247 657164 844044 873001 535057 965357 696796 616349 489402 579162 317053 961928 619512 786770 360648 56465 792898 585083 698277 912525 516345 371280 752078 71383 141404 194980 668092 574093 50763 748353 933713 794633 677515 710502 242437 914652 390217 890730 194169 205038 232939 251657 688727 317165 992437 367151 778435 463983 536785 131848 934599 448083 168836 962293 87746 400849 598191 872286 167684 488701 583652 651690 547812 736996 126665 179891 87823 857225 835317 516443 307877 346323 564746 66875 710012 609983 315794 57133 859688 54917 103585 97512 830699 969743 318901 34790 148312 926533 67980 494334 807836 987861 891558 613713 297340 982465 194444 935555 258596 211287 310631 945182 860270 187338 247655 480595 347075 156087 703882 301569 745628 392306 686241 235207 342126 182761 829459 680553 174449 397184 749765 932913 376060 436326 27481 565539 75894 95612 370825 876474 259792 777715 752784 976518 386153 112948 814339 348630 609652 759089 130019 912614 36542 223843 40441 193735 110027 12644 231433 90245 222222 161318 831011 127775 696695 537980 268819 123294 582612 9191 320410 343076 128547 636329 859147 508346 880348 839883 842422 845923 609378 614787 208701 788777 277127 788555 48629 652046 554497 354666 562693 385327 264012 686226 76244 347599 353701 293372 358691 831862 210111 208458 488893 680231 332993 991178 990747 737443 365792 63142 381550 326998 598164 772030 897188 448960 908816 219465 845968 985654 886801 186082 812280 299008 529952 268067 103024 885690 330103 988180 612247 736274 636532 95298 940272 479125 421403 868492 738167 614014 197806 179994 937633 772940 248402 901083 209404 309151 120538 611799 932373 892559 405540 306935 128839 295086 569353 117708 718235 528085 354613 674980 418996 786369 997492 196170 876104 976069 702902 152753 669947 378146 29222 859398 165326 480374 184236 718687 807744 659796 580114 253599 941876 42400 704910 38560 589302 386190 978740 365881 723311 461877 179925 419659 447906 343510 946448 280292 642650 770777 191374 367136 369200 174983 413388 854632 422579 786669 775770 634621 546504 449756 637020 844075 