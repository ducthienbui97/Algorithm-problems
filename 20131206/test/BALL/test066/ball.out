8419750711240
